routes/events.js

```diff
@@ -35,7 +35,7 @@ router
       }
     })
     .post(needs('events', 'create'), (req, res, next) => {
-      Event.create(req.body, { fields: ['name', 'startDate', 'endDate', 'description', 'location', 'image', 'committeeName'] })
+      Event.create(req.body, { fields: ['name', 'startDate', 'endDate', 'description', 'location', 'link', 'image', 'committeeName'] })
```

Comment:
Snuck this one in, as events can accept link as a field (already exists in the db) and the relevant create event test was failing because this wasn't specified.